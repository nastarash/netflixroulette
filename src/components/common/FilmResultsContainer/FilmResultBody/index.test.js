import React from 'react';
import { FilmResultBody } from './index';

describe('<FilmResultBody />', () => {

  test('should be rendered correctly', () => {
    const filmStub = { 
      Id: 0,
      Image: 'assets/film1.jpg',
      Title: 'First film',
      ReleaseDate: 2000,
      Genre: 'Drama'
    };
    const filmResultBody = shallow(
      <FilmResultBody filmResult={filmStub} />
    );
    expect(filmResultBody).toMatchSnapshot();
  });

});