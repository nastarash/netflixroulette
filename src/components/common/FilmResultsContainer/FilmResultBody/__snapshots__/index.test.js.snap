// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilmResultBody /> should be rendered correctly 1`] = `
<div
  className="film-result-body"
  onClick={[Function]}
>
  <FilmResultImage />
  <div
    className="film-result-body__header"
  >
    <span
      className="film-result-body__title"
    />
    <span
      className="film-result-body__date"
    >
      <FilmResultReleaseDate
        date=""
      />
    </span>
  </div>
  <span
    className="film-result-body__genre"
  />
</div>
`;
